tests/test_qpu.py

```diff
@@ -163,3 +174,82 @@ def circuit():
             return qml.expval(qml.PauliZ(0))
 
         assert np.allclose(circuit(), 0.0, atol=2e-2)
+
+    @flaky(max_runs=10, min_passes=1)
+    def test_2q_gate_pauliz_identity_tensor(self):
+        """Test that the PauliZ tensor Identity observable works correctly.
+
+        As the results coming from the qvm are stochastic, a constraint of 1 out of 10 runs was added.
+        """
+
+        device = np.random.choice(VALID_QPU_LATTICES)
+        dev_qpu = qml.device('forest.qpu', device=device, load_qc=False, readout_error=[0.9, 0.75],
+                            symmetrize_readout="exhaustive", calibrate_readout="plus-eig", shots=QVM_SHOTS)
+
+        @qml.qnode(dev_qpu)
+        def circuit():
+            qml.RY(np.pi/2, wires=[0])
+            qml.CNOT(wires=[0, 1])
+            return qml.expval(qml.PauliZ(0) @ qml.Identity(1))
+
+        assert np.allclose(circuit(), 0.0, atol=2e-2)

```

Comment:
A better test might be a circuit that takes a parameter. For example, 
```python
@qml.qnode(dev_qpu)
def circuit(x):
    qml.RY(x, wires=[0])
    qml.Hadmard(wires=1)
    qml.CNOT(wires=[0, 1])
    return qml.expval(qml.PauliZ(0) @ qml.Identity(1))

assert np.allclose(circuit(a), np.cos(a), atol=2e-2)
```
This could then be parametrized for different values of `a`